<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>152</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>6</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Image Utilities</string>
  </property>
  <widget class="QPushButton" name="pushButton_BrowseAnnotationImages">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>60</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AnnotationImagesFolder">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>55</y>
     <width>710</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="toolTip">
    <string>This field is pre-populated and cannot be changed at this time.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>35</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Training Folder</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox_ok_cancel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>105</y>
     <width>167</width>
     <height>25</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
